demo/mixins.html

```diff
@@ -159,5 +159,33 @@ <h3>Pill Button - ll-button demo</h3>
         </template>
       </demo-snippet>
     </div>
+    <div class="styled vertical-section-container centered">
+      <h3>Disabled Button - ll-button demo</h3>
+      <demo-snippet>
+        <template>
+          <disabled-styled-button></disabled-styled-button>
+          <script type="module">
+            import {LitElement, html} from '../node_modules/@polymer/lit-element/lit-element.js';
+            import {buttonDisabled} from '../ll-button-style-mixins.js';
+            class DisabledStyledButton extends LitElement {
+              render() {
+                return html`
+                    <style>
+                      :host {
+                        ${buttonDisabled}
+                      }
+                    </style>
+                    <ll-button text="Disabled Button Demo"></ll-button>
```

Comment:
Is it overkill to be explicit with this usecase?
```suggestion
                    <ll-button disabled text="Disabled Button Demo"></ll-button>
```
Which would change a bit the styles above as well?